scripts/superdesk-authoring/authoring/controllers/ChangeImageController.js

```diff
@@ -50,7 +50,8 @@ export function ChangeImageController($scope, gettext, notify, modal, $q, _, api
         /* Throw an exception if a required metadata field is missing */
         function validateMediaFields() {
             _.each($scope.data.requiredFields, function (key) {
-                if ($scope.data.metadata[key] == null || _.isEmpty($scope.data.metadata[key])) {
+                var value = $scope.data.metadata[key];
+                if (value == null || _.isEmpty(value) || value === '<br>') {
```

Comment:
Would simply `if (!!!value && value === '<br>')` not do? It should already catch all falsy values (both _null_ and _empty_ string).
Also, what about the other forms of `br`, such as `<br/>`, `<br >`, `<br />`, etc ?